server/src/db/migrations/20230803143105_createReviews.cjs

```diff
@@ -0,0 +1,24 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable("reviews", (table)=>{
+        table.bigIncrements("id")
+        table.string("review").notNullable()
```

Comment:
we tend to want to avoid a column with the same name as the table as it makes it repetitive when we say `review.review` - consider instead body, content, description, etc